### PR TITLE
Eng 9355 geo origin is a point

### PR DESCRIPTION
**Pull request by BillWhite**

### CODE REVIEW

File: tests/frontend/org/voltdb/types/TestPointType.java [hunk 1]
```diff
@@ -29,6 +29,8 @@
 import junit.framework.TestCase;
 
 public class TestPointType extends TestCase {
+    // Points should have this much precision.
+    private final double EPSILON = 1.0e-15;
```
Comment (hhakimi53) [inline tests/frontend/org/voltdb/types/TestPointType.java]: Can the degree of precision change based on what we find out in ENG-9388?

**Comment by hhakimi53** [comment]
Looks good overall

**Comment by BillWhite** [comment]
This was reviewed earlier by Paul.

File: tests/frontend/org/voltdb/types/TestPointType.java (hunk 1 above)
Comment (BillWhite) [inline tests/frontend/org/voltdb/types/TestPointType.java]: Perhaps.  But at least we will have some way of capturing it.

**Pull request merged**